samples/core/src/test/java/io/oasp/gastronomy/restaurant/salesmanagement/service/impl/rest/SalesmanagementHttpRestServiceTest.java

```diff
@@ -61,6 +63,9 @@
   private SalesmanagementRestServiceTestHelper helper;
 
   @Inject
+  private ServiceClientFactory serviceClientFactory;
```

Comment:
My original intent was not to save typing but to delegate object creation to the IoC, whenever I see some code creating objects that could be injected seems a bit odd.

Isn't this more or less the same as Spring Data does with repositories?

I will review your code in order to get more insight into this